Formula/cargo-make.rb

```diff
@@ -0,0 +1,26 @@
+class CargoMake < Formula
+  desc "Rust task runner and build tool"
+  homepage "https://github.com/sagiegurari/cargo-make"
+  url "https://github.com/sagiegurari/cargo-make/archive/refs/tags/0.36.4.tar.gz"
+  sha256 "69c24bd9d0405d07bdf681e4fdf96bf32a95ca4b439f203e7f3997fdf9800781"
+  license "Apache-2.0"
+
+  depends_on "rust" => [:build, :test]
```

Comment:
let's make it as runtime dependency like others?
```suggestion
  depends_on "rust" # uses `cargo` at runtime
```

Reply:
@chenrui333 I have some doubt about this.

There are at least 2 ways to install rust with `brew`: with `rust` directly, or via `rustup-init` (like what I did).

Due to how `cargo-*` binaries work, both approaches should work properly, but now with this change, I also installed `rust` without any practical use at all...


A quick look at other similar binaries like [`cargo-watch`](https://github.com/Homebrew/homebrew-core/blob/17aa4a59969dc7ba12430a84b8b8723ea931a544/Formula/cargo-watch.rb) and [`cargo-udeps`](https://github.com/Homebrew/homebrew-core/blob/17aa4a59969dc7ba12430a84b8b8723ea931a544/Formula/cargo-udeps.rb) further convinced me that this change should not happen. How do you think? 

Many thanks!